docs/pkg.conf.5

```diff
@@ -118,6 +118,9 @@ Default: NO.
 Location where the libraries are backed up.
 Default:
 .Pa /usr/local/lib/compat/pkg .
+.It Cm COMPRESSION_FORMAT: string
+Set the default compression format.
```

Comment:
It would be nice to list the existing supported formats here, as per https://github.com/freebsd/pkg/blob/ad3f9b4902f7e7e8e70ba05e377f49bc63c9b8ae/libpkg/packing.c#L392